### ISSUE TITLE
I can't log into pop os(new cosmic de)

### ISSUE THREAD

**Issue by baaannnaaannn** [pop-os/cosmic-epoch#152]
Can't login into pop os(black screen with Terminal string)![PXL_20231106_225042490.jpg](https://github.com/pop-os/cosmic-epoch/assets/55044749/61474839-b044-48aa-876a-6e534b47298a)

**Comment by jacobgkau** [comment]
What hardware are you using?

After going back to the default GNOME-based DE, can you check `journalctl` for logs at the time you tried to log into COSMIC?

**Comment by baaannnaaannn** [comment]
> `journalctl`

ноя 06 20:31:25 pop-os kernel: Linux version 6.5.4-76060504-generic (jenkins@warp.pop-os.org) (x86_64-linux-gnu-gcc-12 (Ubuntu 12.3.0-1ubuntu>
ноя 06 20:31:25 pop-os kernel: Command line: BOOT_IMAGE=/boot/vmlinuz-6.5.4-76060504-generic root=UUID=2e1ce71e-94ac-440d-82ff-7bc7c06f84ba r>
ноя 06 20:31:25 pop-os kernel: KERNEL supported cpus:
ноя 06 20:31:25 pop-os kernel:   Intel GenuineIntel
ноя 06 20:31:25 pop-os kernel:   AMD AuthenticAMD
ноя 06 20:31:25 pop-os kernel:   Hygon HygonGenuine
ноя 06 20:31:25 pop-os kernel:   Centaur CentaurHauls
ноя 06 20:31:25 pop-os kernel:   zhaoxin   Shanghai  
ноя 06 20:31:25 pop-os kernel: BIOS-provided physical RAM map:
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x0000000000000000-0x000000000009e7ff] usable
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x000000000009f800-0x000000000009ffff] reserved
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x00000000000f0000-0x00000000000fffff] reserved
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x0000000000100000-0x00000000cfedffff] usable
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x00000000cfee0000-0x00000000cfee2fff] ACPI NVS
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x00000000cfee3000-0x00000000cfeeffff] ACPI data
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x00000000cfef0000-0x00000000cfefffff] reserved
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x00000000f0000000-0x00000000f3ffffff] reserved
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x00000000fec00000-0x00000000ffffffff] reserved
ноя 06 20:31:25 pop-os kernel: BIOS-e820: [mem 0x0000000100000000-0x000000016fffffff] usable
ноя 06 20:31:25 pop-os kernel: NX (Execute Disable) protection: active
ноя 06 20:31:25 pop-os kernel: SMBIOS 2.4 present.
ноя 06 20:31:25 pop-os kernel: DMI: Gigabyte Technology Co., Ltd. P35-S3G/P35-S3G, BIOS F3a 12/28/2007
ноя 06 20:31:25 pop-os kernel: tsc: Fast TSC calibration using PIT
ноя 06 20:31:25 pop-os kernel: tsc: Detected 3600.479 MHz processor
ноя 06 20:31:25 pop-os kernel: e820: update [mem 0x00000000-0x00000fff] usable ==> reserved
ноя 06 20:31:25 pop-os kernel: e820: remove [mem 0x000a0000-0x000fffff] usable
ноя 06 20:31:25 pop-os kernel: last_pfn = 0x170000 max_arch_pfn = 0x400000000
ноя 06 20:31:25 pop-os kernel: mtrr: your BIOS has configured an incorrect mask, fixing it.
ноя 06 20:31:25 pop-os kernel: mtrr: your BIOS has configured an incorrect mask, fixing it.
ноя 06 20:31:25 pop-os kernel: mtrr: your BIOS has configured an incorrect mask, fixing it.
ноя 06 20:31:25 pop-os kernel: mtrr: your BIOS has configured an incorrect mask, fixing it.
ноя 06 20:31:25 pop-os kernel: mtrr: your BIOS has configured an incorrect mask, fixing it.
ноя 06 20:31:25 pop-os kernel: mtrr: your BIOS has configured an incorrect mask, fixing it.
ноя 06 20:31:25 pop-os kernel: total RAM covered: 5119M
ноя 06 20:31:25 pop-os kernel: Found optimal setting for mtrr clean up
ноя 06 20:31:25 pop-os kernel:  gran_size: 64K         chunk_size: 1G         num_reg: 6          lose cover RAM: 0G
lines 1-36

**Comment by baaannnaaannn** [comment]
> What hardware are you using?
> 
> After going back to the default GNOME-based DE, can you check `journalctl` for logs at the time you tried to log into COSMIC?

System:
  Host: pop-os Kernel: 6.5.6-76060506-generic x86_64 bits: 64
    Desktop: GNOME 42.5 Distro: Pop!_OS 22.04 LTS
Machine:
  Type: Desktop Mobo: Gigabyte model: P35-S3G v: x.x
    serial: <superuser required> BIOS: Award v: F3a date: 12/28/2007
CPU:
  Info: quad core model: Intel Xeon E5450 bits: 64 type: MCP cache:
    L2: 12 MiB
  Speed (MHz): avg: 3591 min/max: N/A cores: 1: 3591 2: 3600 3: 3600
    4: 3574
Graphics:
  Device-1: AMD Baffin [Radeon RX 550 640SP / 560/560X] driver: amdgpu
    v: kernel
  Display: x11 server: X.Org v: 1.21.1.4 driver: X: loaded: amdgpu,ati
    unloaded: fbdev,modesetting,radeon,vesa gpu: amdgpu
    resolution: 1440x900~60Hz
  OpenGL: renderer: AMD Radeon RX 560 Series (polaris11 LLVM 15.0.7 DRM
    3.54 6.5.6-76060506-generic)
    v: 4.6 Mesa 23.1.3-1pop0~1689084530~22.04~0618746
Audio:
  Device-1: Intel 82801I HD Audio driver: snd_hda_intel
  Device-2: AMD Baffin HDMI/DP Audio [Radeon RX 550 640SP / 560/560X]
    driver: snd_hda_intel
  Sound Server-1: ALSA v: k6.5.6-76060506-generic running: yes
  Sound Server-2: PipeWire v: 0.3.82 running: yes
Network:
  Device-1: Realtek RTL8111/8168/8411 PCI Express Gigabit Ethernet
    driver: r8169
  IF: enp4s0 state: up speed: 100 Mbps duplex: full mac: 00:1d:7d:01:84:29
Drives:
  Local Storage: total: 719.25 GiB used: 192.17 GiB (26.7%)
  ID-1: /dev/sda vendor: AMD Radeon model: R5SL256G size: 238.47 GiB
  ID-2: /dev/sdb vendor: Seagate model: ST500DM002-1SB10A size: 465.76 GiB
  ID-3: /dev/sdc type: USB model: AI Mass size: 15.02 GiB
Partition:
  ID-1: / size: 229.73 GiB used: 184.25 GiB (80.2%) fs: ext4 dev: /dev/sda1
Swap:
  ID-1: swap-1 type: partition size: 4 GiB used: 0 KiB (0.0%) dev: /dev/dm-0
  ID-2: swap-2 type: zram size: 4.79 GiB used: 1.34 GiB (27.9%)
    dev: /dev/zram0
Sensors:
  System Temperatures: cpu: 72.0 C mobo: N/A gpu: amdgpu temp: 29.0 C
  Fan Speeds (RPM): N/A gpu: amdgpu fan: 1225
Info:
  Processes: 292 Uptime: 13h 34m Memory: 4.79 GiB used: 3.41 GiB (71.1%)
  Shell: Bash inxi: 3.3.13

**Comment by baaannnaaannn** [comment]
> After going back to the default GNOME-based DE, can you check `journalctl` for logs at the time you tried to log into COSMIC?

how can i copy this is huge text from the beginning

**Comment by jacobgkau** [comment]
I've edited your previous comment so the output is in a code block. You can do this yourself in the future by adding a line with three backticks (```) before and after the output.

> how can i copy this is huge text from the beginning

You can use `--since YYYY-MM-DD HH:MM:SS` to skip to a certain part (the time when you tried to log in). You can pipe (`|`) the output into `cat` so you can scroll easier, or direct it straight into a file with `> filename.txt` so you can inspect and edit it easier.

Keep in mind that the DE is still in pre-alpha stage and isn't meant for general usage yet.

**Comment by baaannnaaannn** [comment]
after update that i got a few hours ago,de started working

**Comment by Dahipppo1555** [comment]
Got exactly the same symptoms as you did.
only difference is Nvidia RTX 2060s gpu.

**Comment by Dahipppo1555** [comment]
> Can't login into pop os(black screen with Terminal string)![PXL_20231106_225042490.jpg](https://private-user-images.githubusercontent.com/55044749/280867824-61474839-b044-48aa-876a-6e534b47298a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI2MjI3ODAsIm5iZiI6MTcwMjYyMjQ4MCwicGF0aCI6Ii81NTA0NDc0OS8yODA4Njc4MjQtNjE0NzQ4MzktYjA0NC00OGFhLTg3NmEtNmU1MzRiNDcyOThhLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE1VDA2NDEyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiNmM2MDZkMGYwNDJkYTg3OTNkYWYxNDNmYzQ5MzI4ODA0MzY0YjJkYjI0ODVlODU5NTE4ZjI1ODlmNWE4NTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.olVJFL83sAbasRTZIXKwfIDsfpIu-xTjO_jaOXaoSaI)
```

Dec 15 07:33:08 pop-os cosmic-session[6789]: Starting cosmic-session
Dec 15 07:33:08 pop-os cosmic-session[6789]: starting process ' COSMIC_SESSION_SOCK=12 cosmic-comp '
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.190667Z  INFO cosmic_session: Starting cosmic-session
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.190734Z  INFO launch_pad: starting process ' COSMIC_SESSION_SOCK=12 cosmic-comp '
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.199489Z ERROR launch_pad: process ' COSMIC_SESSION_SOCK=12 cosmic-comp ' failed with code 101
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.199500Z ERROR cosmic_session::comp: cosmic-comp exited with error code 101
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.199507Z  INFO launch_pad: draining stdin receiver before restarting process
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.199523Z  INFO launch_pad: sleeping for 5ms before restarting process cosmic-comp (restart 0)
Dec 15 07:33:08 pop-os cosmic-comp[6816]: Failed to read config 'workspaces'
Dec 15 07:33:08 pop-os cosmic-comp[6816]: thread 'main' panicked at 'Malformed config file: SpannedError { code: NoSuchEnumVariant { expected: ["Terminate", "Debug", "Close", "Wo>
                                             0: <backtrace::capture::Backtrace as core::default::Default>::default
                                             1: log_panics::Config::install_panic_hook::{{closure}}
                                             2: <alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call
                                                       at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/alloc/src/boxed.rs:2021:9
                                                std::panicking::rust_panic_with_hook
                                                       at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:711:13
                                             3: std::panicking::begin_panic_handler::{{closure}}
                                                       at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:599:13
                                             4: std::sys_common::backtrace::__rust_end_short_backtrace
                                                       at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/sys_common/backtrace.rs:170:18
                                             5: rust_begin_unwind
                                                       at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:595:5
                                             6: core::panicking::panic_fmt
                                                       at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/panicking.rs:67:14
                                             7: core::result::unwrap_failed
                                                       at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/result.rs:1652:5
                                             8: cosmic_comp::config::Config::load
                                             9: cosmic_comp::state::State::new
                                            10: cosmic_comp::main
                                            11: std::sys_common::backtrace::__rust_begin_short_backtrace
                                            12: main
                                            13: <unknown>
                                            14: __libc_start_main
                                            15: _start
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.206111Z  INFO launch_pad: restarted process ' COSMIC_SESSION_SOCK=12 cosmic-comp ', now at 1 re>
Dec 15 07:33:08 pop-os /usr/libexec/gdm-wayland-session[6789]: 2023-12-15T06:33:08.206145Z  INFO launch_pad: process 'ProcessKey(1v1)' cancelled
Dec 15 07:33:08 pop-os cosmic-session[6789]: process ' COSMIC_SESSION_SOCK=12 cosmic-comp ' failed with code 101
Dec 15 07:33:08 pop-os cosmic-session[6789]: cosmic-comp exited with error code 101
Dec 15 07:33:08 pop-os cosmic-session[6789]: draining stdin receiver before restarting process
Dec 15 07:33:08 pop-os cosmic-session[6789]: sleeping for 5ms before restarting process cosmic-comp (restart 0)
Dec 15 07:33:08 pop-os cosmic-session[6789]: restarted process ' COSMIC_SESSION_SOCK=12 cosmic-comp ', now at 1 restarts
Dec 15 07:33:08 pop-os cosmic-session[6789]: process 'ProcessKey(1v1)' cancelled
Dec 15 07:33:18 pop-os /usr/libexec/gdm-x-session[6445]: dbus-daemon[6445]: [session uid=112 pid=6445] Activating service name='org.freedesktop.systemd1' requested by ':1.8' (uid>
Dec 15 07:33:18 pop-os gsd-print-notif[6519]: Error releasing name org.gnome.SettingsDaemon.PrintNotifications: The connection is closed
```

**Comment by Drakulix** [comment]
Try to delete `$HOME/.config/cosmic/com.system76.CosmicComp`

**Comment by Dahipppo1555** [comment]
> Try to delete `$HOME/.config/cosmic/com.system76.CosmicComp` **and** make sure your config file under `/etc/cosmic-comp/config.ron` matches the current one from this repo. (And that you don't have any overrides in `.config/cosmic-comp.ron` or `.config/cosmic-comp/config.ron`.)
> 
> The error message clearly indicates, that you are running with an outdated config file.

Thanks. error disappeared but i have nvidia gpu. and i read in cosmic-comp that its nvidia's fault it doesnt work.

**Comment by Drakulix** [comment]
nVidia GPUs generally work. What issues do you have currently?

**Comment by Dahipppo1555** [comment]
> nVidia GPUs generally work. What issues do you have currently?

I have only that "_" and black screen.
No idea whats wrong. i tried compiling from source and even .deb package.

I have proprietary 545.29.06, Nvidia 2060s.

**Comment by Drakulix** [comment]
Could you give some more informations about your system?
1. Which distribution are you running?
2. Have you added the `nvidia-drm.modeset=1` kernel flag?
3. How are you trying to launch the session? From a login manager, e.g. GDM? Or from a tty?
4. Could you try running it, reboot and post the logs from the following command please?
`journalctl --user -b-1 _EXE=/usr/bin/cosmic-session`

**Comment by Dahipppo1555** [comment]
> Could you give some more informations about your system?
> 
>     1. Which distribution are you running?
> 
>     2. Have you added the `nvidia-drm.modeset=1` kernel flag?
> 
>     3. How are you trying to launch the session? From a login manager, e.g. GDM? Or from a tty?
> 
>     4. Could you try running it, reboot and post the logs from the following command please?
>        `journalctl --user -b-1 _EXE=/usr/bin/cosmic-session`

1) POP_OS 22.04 LTS

2)  output of /etc/default/grub
`GRUB_CMDLINE_LINUX_DEFAULT="quiet splash acpi_enforce_resources=lax nvidia-drm.modeset=1"`

3) Login manager. Didnt try GDM and tty.

4) log:
```
dahippo1555@pop-os:~$ sudo journalctl --user -b-1 _EXE=/usr/bin/cosmic-session
No journal files were found.
Data from the specified boot (-1) is not available: No such boot ID in journal
```

5) and also. I tried again installing cosmic-* this is from journalctl:
```
Dec 18 13:48:49 pop-os /usr/libexec/gdm-wayland-session[4527]: 2023-12-18T12:48:49.645414Z  INFO cosmic_session: Starting cosmic-session
Dec 18 13:48:49 pop-os /usr/libexec/gdm-wayland-session[4527]: 2023-12-18T12:48:49.645487Z  INFO launch_pad: starting process ' COSMIC_SESSION_SOCK=12 cosmic-comp '
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Failed to read config 'workspaces'
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Key-Binding 'Period' only matched case insensitive for "period"
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Key-Binding 'Comma' only matched case insensitive for "comma"
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Key-Binding 'Period' only matched case insensitive for "period"
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Key-Binding 'Comma' only matched case insensitive for "comma"
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Failed to read config 'xkb-config'
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Failed to read config 'input-default'
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Failed to read config 'input-touchpad'
Dec 18 13:48:49 pop-os cosmic-comp[4554]: Failed to read config 'input-devices'
Dec 18 13:48:50 pop-os cosmic-comp[4554]: Unable to become drm master, assuming unprivileged mode
Dec 18 13:48:50 pop-os cosmic-comp[4554]: [EGL] 0x300c (BAD_PARAMETER) eglQueryDmaBufModifiersEXT: EGL_BAD_PARAMETER error: In eglQueryDmaBufModifiersEXT: Invalid format
Dec 18 13:48:50 pop-os cosmic-comp[4554]: [EGL] 0x300c (BAD_PARAMETER) eglQueryDmaBufModifiersEXT: EGL_BAD_PARAMETER error: In eglQueryDmaBufModifiersEXT: Invalid format
Dec 18 13:48:50 pop-os cosmic-comp[4554]: [EGL] 0x300c (BAD_PARAMETER) eglQueryDmaBufModifiersEXT: EGL_BAD_PARAMETER error: In eglQueryDmaBufModifiersEXT: Invalid format
Dec 18 13:48:50 pop-os cosmic-comp[4554]: [EGL] 0x300c (BAD_PARAMETER) eglQueryDmaBufModifiersEXT: EGL_BAD_PARAMETER error: In eglQueryDmaBufModifiersEXT: Invalid format
```

**Comment by Drakulix** [comment]
Hmm, seems like cosmic-comp is hanging on your system, though it is unclear why that is happening. All log messages are normal, but I would expect the messages to continue or for an image to appear.

**Comment by Dahipppo1555** [comment]
Idk i just today updated flatpak and it works.
No clue what did caused that crash.

**Comment by drazil100** [comment]
I too am having the exact same issue. I installed from the AUR and am getting the exact same journalctl errors and symptoms. I just get a black screen with _ blinking when I try to log in. 

I tried logging into cinnamon Wayland experimental (cinnamon is my main DE) and I get the exact same black screen with blinking _. Plasma Wayland boots just fine though. 

Not sure if a solution was actually found or not yet but figured I'd offer additional clues in case they helped.
